dapps/react-dapp-auth/package.json

```diff
@@ -13,25 +13,27 @@
     "@emotion/react": "^11.10.0",
     "@emotion/styled": "^11.10.0",
     "@walletconnect/auth-client": "2.0.4",
+    "@web3modal/core": "^2.0.0-beta.4",
```

Comment:
Lockfile diff is too big to open for me here but I suspect these are technically also at `2.2.2` in line with `standalone` below?

Could we use the explicit versions for these too instead of the stale beta prereleases to avoid possible confusion?